pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -172,8 +210,27 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 		return
 	}
 
-	// find the list of thresholds that are met independent of grace period
-	now := m.clock.Now()
+	// attempt to create a threshold notifier to improve eviction response time
+	if !m.notifiersInitialized {
+		m.notifiersInitialized = true
+		// start soft memory notification
+		startMemoryThresholdNotifier(m.config.Thresholds, observations, false, func() {
+			glog.Infof("soft memory eviction threshold crossed")
```

Comment:
nit: Can we also include the %tage or absolute value here? It will be helpful while debugging.